pymdownx/progressbar.py

```diff
@@ -4,7 +4,6 @@
 pymdownx.progressbar
 Simple plugin to add support for progress bars
 
-~~~
```

Comment:
Please put the `~~~` back as they are used as fences to prevent the spell checker from checking this region of the docstring.